pkg/manager/agent.go

```diff
@@ -85,6 +87,7 @@ func (a *AgentDaemonSet) Create(image string, managerURL string) (*appsv1.Daemon
 								Capabilities: &corev1.Capabilities{
 									Add: []corev1.Capability{"SYSLOG"},
 								},
+								Privileged: ptr.To(true),
```

Comment:
1. Can we use `hostPID: true` instead of granting full privileges and relying on `nsenter`?
1. This is specific to Longhorn, it should be moved to [prepareDaemonSetForLonghorn](https://github.com/rancher/support-bundle-kit/blob/11e7f1f3f9cc4e606a4c6b2a1912b7155eff7784/pkg/manager/agent.go#L144).

Reply:
Modified. Thanks for the guidance.